src/types/profile-derived.js

```diff
@@ -127,9 +127,14 @@ export type MarkerTiming = {
   index: MarkerIndex[],
   label: string[],
   name: string,
+  bucket: string,
   length: number,
 };
-export type MarkerTimingRows = Array<MarkerTiming>;
+
+export type MarkerTimingBuckets = Array<{
```

Comment:
This is not used. Depending on the answer of my previous question we will need to remove it.